sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcSourceSuite.scala

```diff
@@ -274,4 +274,16 @@ class OrcSourceSuite extends OrcSuite {
       )).get.toString
     }
   }
+
+  test("SPARK-22146: read ORC files containing special characters") {
```

Comment:
Hi, @mgaido91 .
Could we generalize a test case to cover all file-based data sources? Or, at least,for Parquet, too?

cc @gatorsmile

Reply:
Hi @dongjoon-hyun,
thank you for your review.
My only concern is that while ORC support is in the hive module, all the other datasources are in the sql module. So, where should we put the generalized test?

Reply:
For DDL cases, we have HiveDDLSuite extends DDLSuite. But, in this case, I guess maybe HiveQuerySuite? I think the location will be settled during further reviews.

Reply:
You can do it in `MetastoreDataSourcesSuite`